packages/perseus/src/widgets/__tests__/radio_test.jsx

```diff
@@ -224,12 +231,14 @@ describe("single-choice question", () => {
             renderQuestion(q, apiOptions);
 
             // Act
-            userEvent.tab();
-            userEvent.tab();
-            userEvent.tab();
+            userEvent.tab(); // first screen reader radio
+            userEvent.tab(); // first button
+            userEvent.tab(); // second screen reader radio
+            userEvent.tab(); // second button
+            userEvent.tab(); // third screen reader radio
```

Comment:
Visual users will have to tab twice to move between options. I don't think Jeanette has had a chance to look at the tests as I worked on these on my own. Since perseus-sr-only only visually hides options, it is still on the page. If there were some way to hide the SR content from tabbing when not in VoiceOver, that would prevent this. I feel like anything that would prevent it from being tab-able would not be able to tell if VoiceOver (or another screen reader) was on though.

Reply:
As someone who relies heavily on keyboard navigation myself, I would consider this a blocker to merging. Could you find and propose a solution? You don't have to implement it, maybe just an idea on how we could avoid this problem.